QModManager/Patcher.cs

```diff
@@ -144,7 +224,19 @@ internal static void LoadMods()
                 if (mod != null && !mod.Loaded && mod.Id != "SMLHelper")
                 {
                     toWrite += $"- {mod.DisplayName} ({mod.Id})\n";
-                    loadedMods.Add(LoadMod(mod));
+
+                    if(!LoadMod(mod))
+                    {
+                        if (!erroredMods.Contains(mod))
+                            erroredMods.Add(mod);
+
+                        if (!loadingErrorMods.Contains(mod))
+                            loadingErrorMods.Add(mod);
+
+                        continue;
+                    }
+
+                    loadedMods.Add(mod);
                 }
                 else if(mod != null && !mod.Loaded && mod.Id == "SMLHelper")
```

Comment:
`mod != null && !mod.Loaded` is already checked in the if above. This could be changed to:

    if (mod != null && !mod.Loaded)
    {
        if (mod.Id == "SMLHelper")
        {
            ...
        }
        else
        {
            ...
        }
    }